licensing/templates/main-menu.php

```diff
@@ -0,0 +1,159 @@
+
+<?php
+/**
+ * Module-specific PHP for creating the main menu links in #main-menu-titles
+ * Included in ../templates/title.php
+ */
+
+
+if ($user->isAdmin()) { ?>
+
+    <a href='index.php'>
+        <div class="main-menu-link <?php if ($currentPage == 'index.php') { echo "active"; } ?>">
+            <img src="images/menu/icon-home.png" />
```

Comment:
According to the W3C Validator these are invalid because “required attribute ‘alt’ not specified”. We should probably do this with CSS background images anyway.